Add content for eBPF documentation (#12417)
netdata/netdata: collectors/ebpf.plugin/README.md

### Disk

The eBPF plugin also shows a chart in the Disk section when the `disk` thread is enabled.

#### Disk Latency
This will create the chart `disk_latency_io` for each disk on the host. The following tracepoints are used:

- [`block/block_rq_issue`](https://www.kernel.org/doc/html/latest/core-api/tracepoint.html#c.trace_block_rq_issue):
  IO request operation to a device drive.
- [`block/block_rq_complete`](https://www.kernel.org/doc/html/latest/core-api/tracepoint.html#c.trace_block_rq_complete):
  IO operation completed by device.

Disk Latency is the single most important metric to focus on when it comes to storage performance, under most circumstances.
For hard drives, an average latency somewhere between 10 to 20 ms can be considered acceptable. For SSD (Solid State Drives),

> depending on the workload it should never reach higher than 1-3 ms. In most cases, workloads will experience less than

Reviewer: ```suggestion
depending on the workload it should never reach higher than 1-3 ms. In most cases, workloads will experience less than
```

I'm going to suggest we combine this sentence and the next one to say "In most cases, workloads experience less than 1 ms latency numbers, but workloads should never reach higher than 3 ms."

1ms latency numbers. The dimensions refer to time intervals.

### Filesystem

This group has charts demonstrating how applications interact with the Linux kernel to open and close file descriptors.
It also brings latency charts for several different filesystems.

#### Latency Algorithm

> We calculate the difference between the calling and return times, this spans disk I/O, file system operations (lock, I/O),

Reviewer: ```suggestion
We calculate the difference between the calling and return times, spanning disk I/O, file system operations (lock, I/O),
```

- [`d_lookup`](https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/fs/dcache.c?id=052b398a43a7de8c68c13e7fa05d6b3d16ce6801#n2223):
  Called when the desired file is not inside the directory cache.

##### Directory Cache Interpretation

> When directory cache is showing 100% means that every file that was accessed was present in the directory cache.

Reviewer: ```suggestion
When directory cache is showing 100%, that means that every file that was accessed was present in the directory cache.
```

> When directory cache is showing 100% means, that means that every file that was accessed was present in the directory cache.

Reviewer 2: ```suggestion
When directory cache is showing 100% that means that every accessed file was present in the directory cache.
```

> If files are not present in the directory cache 1) they are not present in the file system, 2) the files were not

Reviewer: Is 1 and 2 an "and" or "or" situation?

> If files are not present in the directory cache they are not present in the file system, or the files were not

Reviewer: ```suggestion
If files are not present in the directory cache, they are either not present in the file system or the files were not
```
